src/nxt_conf_validation.c

```diff
@@ -1786,6 +1801,28 @@ nxt_conf_vldt_python_protocol(nxt_conf_validation_t *vldt,
 }
 
 
+static nxt_int_t
+nxt_conf_vldt_python_prefix(nxt_conf_validation_t *vldt,
+    nxt_conf_value_t *value, void *data)
+{
+    nxt_str_t  prefix;
+
+    if (nxt_conf_type(value) != NXT_CONF_STRING) {
+        return nxt_conf_vldt_error(vldt, "The \"prefix\" must be a string "
+                                   "beginning with \"/\".");
```

Comment:
I think I prefer specifying the character as `'/'`, instead of `\"/\"`, for consistency with C's syntax for referring to strings and characters.

Reply:
The value of `"prefix"` is set in JSON, where only double-quotes are allowed. I was more worried about consistency with the language of the configuration (JSON) than the language of the program validating the configuration (C).

I can change it if you want.

Reply:
Hmm, I don't have a strong opinion here.  You opinion makes sense too.  I didn't think of JSON, because as it doesn't represent a full JSON string, but rather one character of a JSON string...  But whatever makes more sense to you is good.

So, you decide.

Reply:
I think I will keep it as-is. We shouldn't assume that the developers using this application are all familiar with C and the difference between double- and single-quotes.